plugins/drive/main.go

```diff
@@ -56,5 +56,5 @@ func (d *destination) Configure(config map[string]interface{}) error {
 
 func main() {
 	reg := prometheus.NewRegistry()
```

Comment:
i see. we should just remove the option. It does not do anything. I guess we want to use the default registry here. Otherwise we would not have any metrics for the plugin